### PR TITLE
chore: add volunteer rahulpandey70

### PR DESCRIPTION
**Pull request by rahulpandey70**
add volunteer

### CODE REVIEW

File: data/users/rahulpandey70.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Rahul Pandey",
+  "bio": "Full Stack Developer",
+  "image": "https://avatars.githubusercontent.com/u/34403988?s=400&u=733d087b149c2e488b69a5e41e63aae242b31c69&v=4",
```
Comment (codewithvoid) [inline data/users/rahulpandey70.json:4]: use simpler image url. e.g
```json
"image": "https://avatars.githubusercontent.com/u/34403988?v=4"
```

File: data/users/rahulpandey70.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      "slug": "web-development",
+      "topics": ["ReactJS", "NodeJS", "ExpressJS"]
+    },
+    {
+      "slug": "database-development",
+      "topics": ["MongoDB"]
+    },
+    {
+      "slug": "devops",
+      "topics": ["Docker", "Kubernetes", "Aws"]
```
Comment (codewithvoid) [inline data/users/rahulpandey70.json:20]: nit: `AWS`

**Review by codewithvoid — CHANGES_REQUESTED** [review]
Thanks @rahulpandey70 for volunteering. Change looks great. Added small comments. Please check.

File: data/users/rahulpandey70.json (hunk 1 above)
Comment (rahulpandey70) [reply data/users/rahulpandey70.json]: done

**Review by codewithvoid — APPROVED** [review]
Perfect. Welcome!

You can join the community discord [here](https://discord.gg/Tn9E5pn5)

**Pull request merged**